src/promptflow-devkit/promptflow/_cli/_pf/_flow.py

```diff
@@ -510,18 +511,21 @@ def _test_flow_multi_modal(args, pf_client):
         from promptflow._sdk._tracing import _invoke_pf_svc
 
         # Todo: use base64 encode for now, will consider whether need use encryption or use db to store flow path info
-        def generate_url(flow_path, port):
+        def generate_url(flow_path, port, enable_internal_features=False):
```

Comment:
will this PR also address case that without --ui?

Reply:
yeap